src/containers/Tenant/Diagnostics/TenantOverview/TenantStorage/TopGroups.tsx

```diff
@@ -46,9 +44,7 @@ export function TopGroups({tenant}: TopGroupsProps) {
 
     const loading = isFetching && currentData === undefined;
 
-    const preparedGroups = React.useMemo(() => {
-        return prepareTopStorageGroups(currentData);
-    }, [currentData]);
```

Comment:
Backend limit and sort works fine on `ydb-stable-24-1-14` and after, so limiting and sorting processing is redundant